tests/foreman/api/test_reporttemplates.py

```diff
@@ -666,7 +666,7 @@ def test_positive_schedule_entitlements_report(setup_content, target_sat):
 
 @pytest.mark.no_containers
 @pytest.mark.tier3
-def test_positive_generate_job_report(setup_content, target_sat, rhel7_contenthost):
+def test_positive_generate_job_report(setup_content, target_sat, content_hosts):
```

Comment:
`setup_content` fixture uses `module_target_sat`, can you make all the tests more consistent by using it too? This can avoid some unwanted issues.
```suggestion
def test_positive_generate_job_report(setup_content, module_target_sat, content_hosts):
```